src/Klarna/Rest/MerchantCardService/VCCSettlements.php

```diff
@@ -137,7 +140,9 @@ public function retrieveOrderSettlement($orderId, $keyId)
             'GET',
```

Comment:
Use constant of Method class instead of string

Reply:
Replaced all the text methods in all test files.